src/modules/components/GoogleMapsCard.tsx

```diff
@@ -0,0 +1,97 @@
+import { Box, Center, Spinner } from "@chakra-ui/react";
+import {
+  GoogleMap,
+  useLoadScript,
+  DirectionsService,
+  DirectionsRenderer,
+  DirectionsServiceProps,
+} from "@react-google-maps/api";
+import { useRef, useState } from "react";
+
+interface Props {
+  data?: any;
```

Comment:
Maybe the type of `data` will be found in [`@types/google-map-react`](https://www.npmjs.com/package/@types/google-map-react)

Reply:
[chore: correct Google Maps DirectionService type](https://github.com/CoRide-tw/frontend/pull/2/commits/d21a95436c9b8b990a9de47f083b684809cbe72a) fixed!!